modules/rostests/apitests/utildll/CachedGetUserFromSid.c

```diff
@@ -0,0 +1,102 @@
+/*
+ * PROJECT:     ReactOS API Tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Tests for CachedGetUserFromSid
+ * COPYRIGHT:   Copyright 2020 Serge Gautherie <reactos-git_serge_171003@gautherie.fr>
+ */
+
+#include <wchar.h>
+
+#include <apitest.h>
+
+static VOID (WINAPI *pCachedGetUserFromSid)(PSID pSid, LPWSTR pUserName, PULONG pcwcUserName);
+
+static void test_Sid_Null(void)
+{
+    const WCHAR szUserNameFromNull[] = L"(unknown)";
+    const ULONG cchUserNameFromNull = (ULONG)wcslen(szUserNameFromNull);
+
+    WCHAR szUserName[MAX_PATH];
```

Comment:
User names aren't paths. `UNLEN + 1` would be better suited.

Reply:
Right. Replaced.